aws/resource_aws_spot_fleet_request_test.go

```diff
@@ -1511,6 +1511,87 @@ resource "aws_spot_fleet_request" "foo" {
 `, rName, rInt, rInt, rName)
 }
 
+func testAccAWSSpotFleetRequestConfigWithoutSpotPrice(rName string, rInt int) string {
```

Comment:
This PR is in great shape -- only item is that this new test configuration does not look like its being called. If you fix that up then we should be able to get this in 👍